bundle/src/main/java/com/jfrog/ide/eclipse/scan/GradleScanManager.java

```diff
@@ -57,39 +60,17 @@ public static boolean isApplicable(IProject project) {
 
 	@Override
 	void refreshDependencies(IProgressMonitor monitor) throws IOException {
```

Comment:
If all the code is here was removed - why do we need this function?

Reply:
Removed function definition as abstract in the base class, since its implementation is no longer mandatory.